operate/client/src/App/Processes/MigrationView/BottomPanel/index.test.tsx

```diff
@@ -648,6 +667,6 @@ describe('MigrationView/BottomPanel', () => {
     await user.click(screen.getByLabelText(/show only not mapped/i));
 
     // Expect all rows to be visible again
-    expect(await screen.findAllByRole('row')).toHaveLength(32);
+    expect(await screen.findAllByRole('row')).toHaveLength(34);
```

Comment:
I know I fixed this in another PR. This PR is probably outdated, so I will fix this when I rebase the branch